ksqldb-common/src/test/java/io/confluent/ksql/schema/OperatorTest.java

```diff
@@ -95,11 +96,11 @@ public void shouldResolveModulusReturnType() {
   @Test
   public void shouldWorkUsingSameRulesAsBaseTypeUpCastRules() {
     allOperations().forEach(op -> {
-
-      for (final SqlBaseType leftBaseType : SqlBaseType.values()) {
+      SqlBaseType[] types = ArrayUtils.removeElement(SqlBaseType.values(), SqlBaseType.LAMBDA);
```

Comment:
This logic is definitely messy - the goal is to keep the test the same for when we add adiditonal sql types but ignore lambdas as we don't need full functionality for them. The lambda sql type is meant to be an internal type so is not so relevant to many of these tests

Reply:
Why do we need this? Looking at the logic of the test, `shouldBeSupported` should return `false` here and we should get the exceptions we expect - right?